src/platform/nrfconnect/NetworkCommissioningDriver.h

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
Consider naming the file after the class name it defines - `NrfWifiDriver.h`. Will be easier to find it.

Reply:
I agree. I just wanted to keep the convention from other platforms.